### ISSUE TITLE
Use staticmethod() as a decorator

### ISSUE THREAD

**Issue by newnone** [claird/PyPDF4#8]
Up until PyPDF2 it is very frequent to see such code snippets as:
```
def method(arg1, arg2, ...):
    ...

method = staticmethod(method)
```

within the body of a class. If we set out to support only down to Python 2.7, then we can substitute those instructions with the more concise
```
@staticmethod
def method(arg1, arg2, ...):
    ...
```

The `staticmethod()` feature was added in Python 2.2, and the decorator support in 2.4: [source](https://docs.python.org/2.7/library/functions.html#staticmethod). Apparently, we are good to go. Shall I add this change into one of my future commits?

**Comment by newnone** [comment]
Also, what should be the style of the future invocations to `staticmethod()`?

**Comment by newnone** [comment]
This is being tackled and closed definitely by PR #9.

**Comment by claird** [comment]
Oscar, it's time to leave 2.6 behind.  Feel free to decorate in a more
natural style.

Cameron Laird, vice president
We make computers work for people.
